core-rust/core-api-server/src/core_api/constants.rs

```diff
@@ -1,4 +1,5 @@
 pub(crate) const MAX_STREAM_COUNT_PER_REQUEST: u16 = 10000;
+pub(crate) const MAX_STREAM_TOTAL_SIZE_PER_RESPONSE: usize = 40 * 1024 * 1024;
```

Comment:
Note: I am not 100% on this limit. Can the (JSON) receipt be bigger than this? A bigger limit seems too loose, but obviously we want to be able to return at least 1 entry :)

Reply:
I think that special-casing "return >=1" is very valid anyways 👍

Reply:
I agree with Jakub, and relatedly - I don't think this needs to be a hard-and-fast limit, and we should give a unit.

Perhaps a `STREAM_CAP_RESPONSE_WHEN_ABOVE_BYTES` or something :)